code/parachain/frame/cosmwasm/src/runtimes/wasmi.rs

```diff
@@ -102,12 +102,14 @@ pub enum InitialStorageMutability {
 }
 
 /// VM shared cache
+#[derive(Clone)]
 pub struct CosmwasmVMCache {
 	/// Code cache, a mapping from a CosmWasm contract code id to the baking code.
 	pub code: BTreeMap<CosmwasmCodeId, Vec<u8>>,
 }
 
 /// VM shared state
+#[derive(Clone)]
```

Comment:
will it clone whole wasm module if it is loaded into?  is it too risky to miss such code in production?